apps/drupal/templates/navigation/menu--account.html.twig

```diff
@@ -0,0 +1,48 @@
+{#
+/**
+ * @file
+ * Theme override to display a menu.
+ *
+ * Available variables:
+ * - menu_name: The machine name of the menu.
+ * - items: A nested list of menu items. Each menu item contains:
+ *   - attributes: HTML attributes for the menu item.
+ *   - below: The menu item child items.
+ *   - title: The menu link title.
+ *   - url: The menu link url, instance of \Drupal\Core\Url
+ *   - localized_options: Menu link localized options.
+ *   - is_expanded: TRUE if the link has visible children within the current
+ *     menu tree.
+ *   - is_collapsed: TRUE if the link has children within the current menu tree
+ *     that are not currently visible.
+ *   - in_active_trail: TRUE if the link is in the active trail.
+ */
+#}
+{% import _self as menus %}
+
+{#
+  We call a macro which calls itself to render the full tree.
+  @see http://twig.sensiolabs.org/doc/tags/macro.html
+#}
+{{ menus.menu_links(items, attributes, 0) }}
```

Comment:
:HEADEXPLODE:

Reply:
This is direct from stable. I think these can be more extensively refactored once @rloos289 nav work is brought in.

Reply:
Since this is straight from Stable, let's get some comments about the Particle-specific additions to the code, and what those additions do.

Reply:
Noting here that this is pretty low touch, dont' need a lot of detail